### DISCUSSION TITLE
MHI2_ER_SEG11_P4193_MU1146 Patch?

### DISCUSSION THREAD

**Discussion by critical3rr0r** [Mr-MIBonk/M.I.B._More-Incredible-Bash#87] [Q&A] [answered]
Hi,
my Seat Leon 5F has the above mentioned firmware installed. The firmware is already available at mibsolution.one, but there is no patch for it. Is it possible to create a patch for this version? Or should I upload the backup first?

**Comment by harman-f** [comment, marked as answer]
yes, upload a backup :)

**Reply by critical3rr0r** [reply]
https://mibsolution.one/index.php/f/176565

It is in the M.I.B_Backup folder.

**Reply by harman-f** [reply]
found it

**Reply by harman-f** [reply]
Patch is same like MHI2_ER_SKG13_P4177_MU1125_PATCH --> MU1125-ifs-root-part2-0x00ba0000-44E0F200.ifs

rename that folder and file like this:
![image](https://user-images.githubusercontent.com/98130152/211221561-60050eb6-14ad-46df-8d6d-2b77e6d3ebae.png)

**Reply by harman-f** [reply]
also take addFecs.txt from a SEG11 folder

**Reply by harman-f** [reply]
you might also have a look at mibsolution.one

**Reply by critical3rr0r** [reply]
Okay, thanks. I found the patch zip and will try it later :)

**Reply by critical3rr0r** [reply]
Ok, everything seems to have worked. When patching, the message like in the picture appeared towards the end. Hope it is not dramatic😇
![F69F0504-50FA-4CD2-B38B-9C5111E107FA](https://user-images.githubusercontent.com/122155842/211311296-59ac71d9-72b6-4341-9c2d-7636d58cdbf8.jpeg)

**Reply by harman-f** [reply]
That is like it should be :)

**Comment by critical3rr0r** [comment]
ok, done :)
